app/mailers/user_mailer.rb

```diff
@@ -13,4 +14,22 @@ def sign_in_email(user, url)
       },
     )
   end
+
+  def confirmation_instructions(user, token, _options = {})
```

Comment:
Do we need the `_options` argument in here?

Reply:
Yes, devise requires it and provides the options param when it calls this method. Taking it out will cause an error